microsoft/testsuites/network/ethtool_suite.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from assertpy import assert_that
+
+from lisa import Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.tools import Ethtool
+
+
+@TestSuiteMetadata(
+    area="network",
+    category="functional",
+    description="""
+    This test suite runs the ethtool related network test cases.
+    """,
+)
+class ethtool(TestSuite):
```

Comment:
It looks this test suite should be called ringbuffer, not ethtool. The class name should be Camel case.

Reply:
If it will contain other network test cases, it may be called Networking?

Reply:
It will contain many other ethtool related test cases which tests different capabilities and features of ethtool.

Reply:
Yes. It uses "ethtool" to test networking, not test "ethtool" itself.

Reply:
> Yes. It uses "ethtool" to test networking, not test "ethtool" itself.

It uses ethtool to test some features offered by ethtool to change networking settings, performance measurements etc

Reply:
It may be confusing to users of LISA test suites. The test suite is to test "networking settings" by "ethtool". The tested object is the networking settings. So it's a test suite about "networking settings", not a test suite about "ethtool". In some cases, we may need other tools to check the changes, which are made by ethtool.